cmd/buildah/export.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/containers/storage/pkg/archive"
+	"github.com/pkg/errors"
+	"github.com/projectatomic/buildah"
+	"github.com/urfave/cli"
+)
+
+var (
+	exportFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "output, o",
+			Usage: "write to a file, instead of STDOUT",
+		},
+	}
+	exportCommand = cli.Command{
+		Name:  "export",
+		Usage: "Export the contents of a container's filesystem as a tar archive",
+		Description: `Export the contents of a container's filesystem using the full or shortened container ID or container name. The output is exported to STDOUT and can be redirected to a tar file.
+
+   Stream to a file instead of STDOUT by using -o.
```

Comment:
Cleaned it up to fix better in an 80 Char tty.

```
 ./buildah export --help
NAME:
   buildah export - Export container's filesystem contents as a tar archive

USAGE:
   buildah export [command options] CONTAINER

DESCRIPTION:
   Export the contents of a container's filesystem using the full or shortened 
   container ID or container name. The output is exported to STDOUT and can be 
   redirected to a tar file.

   Stream to a file instead of STDOUT by using -o.
       

OPTIONS:
   --output value, -o value  write to a file, instead of STDOUT
```